src/Braintree/Braintree.csproj

```diff
@@ -45,10 +44,19 @@
     <DefineConstants>$(DefineConstants);netcore</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <DefineConstants>$(DefineConstants);netcore</DefineConstants>
+  </PropertyGroup>
+
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
```

Comment:
Would you mind just combining this with the netstandard1.3 ItemGroup if there's no difference in the versions?

Reply:
@jackellenberger combined the itemgroups with new commit